pkg/agent/core/ngt/service/ngt.go

```diff
@@ -456,6 +459,12 @@ func (n *ngt) loadKVS(path string) (err error) {
 		return err
 	}
 
+	if n.kvs == nil {
+		n.kvs = kvs.New(kvs.WithConcurrency(n.kvsdbConcurrency))
+	} else if n.kvs.Len() > 0 {
+		n.kvs.Close()
+		n.kvs = kvs.New(kvs.WithConcurrency(n.kvsdbConcurrency))
```

Comment:
Why does existing KVS trash in this moment?

Reply:
The loadKVS function is called at startup. In CoW, loadKVS is called in the order of origin, old, base, and tmp, but for each load, the previous load result may remain in kvs, so it is flushed.